packages/@react-spectrum/menu/test/MenuTrigger.test.js

```diff
@@ -250,14 +250,14 @@ describe('MenuTrigger', function () {
       Name             | Component      | props
       ${'MenuTrigger'} | ${MenuTrigger} | ${{}}
     `('$Name autofocuses the selected item on menu open', function ({Component, props}) {
-      let tree = renderComponent(Component, props, {selectedKeys: ['Bar']});
+      let tree = renderComponent(Component, props, {selectedKeys: ['Bar'], selectionMode: 'single'});
```

Comment:
https://github.com/adobe/react-spectrum/commit/4e85079d656cec366056c2ff06f7731df07def44 for context. Essentially opening the first submenu in [that old story](https://reactspectrum.blob.core.windows.net/reactspectrum/82c23dbc122a23d05f2ba3e3968d45d36d82fc2b/storybook/index.html?path=/story/menutrigger-submenu--sub-menu-selection&providerSwitcher-express=false) via keyboard was autofocusing `Lvl 2 Item 3` since it is the only key in the selectedKey array, but it felt weird since it isn't actually a selectable key since it is a submenu trigger menu item. That led to the change in [useSelectableCollection](https://github.com/adobe/react-spectrum/commit/4e85079d656cec366056c2ff06f7731df07def44#diff-e356ae602508922357fd9cd1aa7896f74e24f87bcb2ee86a7bb4a486ea87a2f2R384-R388) where focus only get will get moved to an actually valid selected key, thus making it so I needed to change this test so that it actually supports selection or otherwise focus wouldn't get moved to "Bar"